extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/media/GoogleMediaExporter.java

```diff
@@ -435,18 +451,29 @@ private MediaContainerResource convertMediaListToResource(
 
       if (mediaItem.isPhoto()) {
         if (shouldUpload) {
-          PhotoModel photoModel = GoogleMediaItem.convertToPhotoModel(albumId, mediaItem);
-          photos.add(photoModel);
-
-          monitor.debug(
-              () -> format("%s: Google exporting photo: %s", jobId, photoModel.getDataId()));
+          try {
+            PhotoModel photoModel = GoogleMediaItem.convertToPhotoModel(albumId, mediaItem);
+            photos.add(photoModel);
+
+            monitor.debug(
+                () -> String.format("%s: Google exporting photo: %s", jobId, photoModel.getDataId()));
+          } catch(ParseException e) {
+            monitor.debug(() -> "Parse exception occurred while converting photo, skipping this item. "
```

Comment:
Wait so do we need these to be monitor.info? I think monitor.debug logs aren't permanently logged, which is why we have job ids and data ids in them. @jzacsh if you could weigh in

```suggestion
            monitor.info(() -> "Parse exception occurred while converting photo, skipping this item. "
```